modules/nw-service-externalip-create.adoc

```diff
@@ -6,70 +6,109 @@
 [id="nw-service-externalip-create_{context}"]
 = Attaching an ExternalIP to a service
 
-You can attach an ExternalIP to a service. If your cluster is configured to allocate an ExternalIP automatically, you might not need to manually attach an ExternalIP to the service.
+You can attach an ExternalIP resource to a service. If you configured your cluster to automatically attach the resource to a service, you might not need to manually attach an ExternalIP to the service.
+
+The examples in the procedure use a scenario that manually attaches an ExternalIP resource to a service in a cluster with an IP failover configuration. 
 
 .Procedure
 
-. Optional: To confirm what IP address ranges are configured for use with ExternalIP, enter the following command:
+. Confirm compatible IP address ranges for the ExternalIP resource by entering the following command in your CLI:
 +
 [source,terminal]
 ----
 $ oc get networks.config cluster -o jsonpath='{.spec.externalIP}{"\n"}'
 ----
 +
-If `autoAssignCIDRs` is set, {product-title} automatically assigns an ExternalIP to a new `Service` object if the `spec.externalIPs` field is not specified.
+[NOTE]
+====
+If `autoAssignCIDRs` is set and you did not specify a value for `spec.externalIPs` in the ExternalIP resource, {product-title} automatically assigns ExternalIP to a new `Service` object.
+====
 
-. Attach an ExternalIP to the service.
+.. Get the YAML manifest for the IP failover deployment by entering the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get deployments -n ipfailover -o yaml | grep <HAproxy_VIP> -A 1 <1>
```

Comment:
the project name "ipfailover" in this case is implementation dependent, so I don't think we want to name it explicitly as in this example.  I think we would leave this series of addtions out (line 27-31).  If a customer deployed an ip failover solution they should know how to get the ip's it is hosting.

Reply:
OK. Thanks. I removed:

````
.. Get the YAML manifest for the IP failover deployment by entering the following command:
+
[source,terminal,subs="attributes+"]
----
$ oc get deployments -n ipfailover -o yaml | grep <HAproxy_VIP> -A 1 <1>
----
<1> Replace `<HAproxy_VIP>` with the single Virtual IP address of the backend server in your HAproxy-configured cluster. Approved external hosts can connect to the service by using this VIP address. 
+
.Example output
[source,terminal,subs="attributes+"]
----
 - name: <HAproxy_VIP>
   value: 192.168.123.22,192.168.123.25
----
````